### PR TITLE
Update overall info for authors to include information on article style and layout

### PR DESCRIPTION
**Pull request by davidlmobley**
Adds guidelines on writing style ( #38 ) etc., and also info on ensuring the final paper looks good as formatted.

### CODE REVIEW

**Comment by davidlmobley** [comment]
@mrshirts , review?

File: _policies/01_policies.md [hunk 1]
```diff
@@ -59,6 +59,34 @@ Additionally, articles should include clear links to, and mention of, the releva
...
+You may receive some comments from your editor and/or the peer reviewers that point out typos or other issues, but you should not **rely** on this for your editing process. 
+In order to keep costs to authors at a minimum, we do not employ copy editors, so be sure to arrange for your own editing.
+
+Articles can have any length; however, you should be as concise as possible.
+
+### Article layout
+
+You are responsible for preparing your article PDF and materials in the way you want them to appear in LiveCoMS, so it is important to take some care as to how these look.
+LiveCoMS will not be separately typesetting your article for you, so be careful that your materials are laid out well.
+Be sure to pay particular attention to:
```
Comment (mrshirts) [inline _policies/01_policies.md]: Probably want to emphasize that we are providing article templates, and that articles being submitted should use these templates.

**Comment by mrshirts** [comment]
See comment above to address.  Otherwise LGTM.

File: _policies/01_policies.md (hunk 1 above)
Comment (davidlmobley) [reply _policies/01_policies.md]: Thanks. I added a link to this effect. This is just shortly following the material where we highlight how everyone needs to use templates, so we don't need to say too much here or we'll be repetitive (but some repetition is probably a good thing).

**Pull request merged**